src/main/java/frc/team1816/robot/LogThread.java

```diff
@@ -0,0 +1,52 @@
+package frc.team1816.robot;
+
+import badlog.lib.BadLog;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Threads;
+import edu.wpi.first.wpilibj.Timer;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class LogThread extends Thread {
+    private BadLog logger;
+
+    public LogThread() {
+        super();
+        // Reduce the priority of this thread
```

Comment:
may want to change this to setDaemon(true).  This should set the thread to the lowest priority background thread and allow the JVM to shutdown without worrying about the state of this thread.

Reply:
Good idea.

Reply:
However, the `Threads` class is a class provided by WPILib to work specifically with the RoboRIO. I'm worried changing this behavior would break it or cause issues.